### ISSUE TITLE
Checkboxes not working on version 2.0.2

### ISSUE THREAD

**Issue by emcga** [adarshpastakia/aurelia-ui-framework#43]
@adarshpastakia It seems that with version 2.0.2 the checkboxes will not check anymore. I made sure all sass files were up to date. Are you experiencing the same issue or is it my configuration only? Thank you.

**Comment by emcga** [comment]
@adarshpastakia If I bind the checkbox's checked property to a property in my view-model, and set it true, the checkbox is checked when the browser content is reloaded. Maybe the event is being canceled before it hits the framework code? Just a guess, I haven't looked at your code yet to see how everything is wired. Again, thanks for your time on this issue.

**Comment by adarshpastakia** [comment]
@emcga i tried the demo at http://adarshpastakia.github.io/aurelia-ui-framework/#/home, the checkboxes work fine. can you post a snippet of your code, im not sure what the issue is exactly

**Comment by emcga** [comment]
Sure, I put together a small demo that has the same checkbox issue.

check_test.js
```javascript
export class CheckTest{

  constructor() {
    this.checked_prop = false;
  }
}
```

check_test.html
```html
<template>
  <ui-page page-title="Check Test">
    <ui-section>
      <ui-content class="ui-flex-column" scroll>
        <ui-checkbox checked.bind="checked_prop">Check Box</ui-checkbox>
      </ui-content>
    </ui-section>
  </ui-page>
</template>
``` 

If I set checked_prop = true the checkbox renders a check mark. But clicking on the checkbox does not change its state. I was trying to trace the code to see if I could find the issue, but debugging JS is not like Java or C++, which are the languages i'm most proficient with.

Also, I wanted to add that the JS console in my browser is not displaying errors and everything else seems to be working fine. I tried IE, Firefox, and Chrome.

I will continue to try to sort things out. If it's working for you then it has to be on my end somewhere. 

Thanks

**Comment by emcga** [comment]
The interesting thing is this issue just started after I did a "jspm update" this morning.

**Comment by emcga** [comment]
@adarshpastakia I cloned the aurelia-ui-framework repo and everything worked fine until I ran "jspm update." When I ran the update the checkbox issue was duplicated. I'm assuming one of the packages in the update is causing the issue. I do not have time to determine what package is causing the issue at the moment, but I will check it out tomorrow.

**Comment by adarshpastakia** [comment]
@emcga i have updated a fix for the checkboxes, however on radio input although the value is being bound properly the check mark issue still exists

**Comment by emcga** [comment]
Great! Thank you.

**Comment by emcga** [comment]
Great! Thank you. I'm not using radio buttons at the moment, but I will be in the next month.

**Comment by adarshpastakia** [comment]
Fixed in version 2.0.5
